src/main/io/serial_cli.c

```diff
@@ -1091,19 +1091,15 @@ static bool isEmpty(const char *string)
     return *string == '\0';
 }
 
-static void printRxFail(uint8_t dumpMask, const rxConfig_t *defaultRxConfig)
+static void printRxFail(uint8_t dumpMask, const rxConfig_t *rxConfig, const rxConfig_t *defaultRxConfig)
 {
     // print out rxConfig failsafe settings
-    const rxFailsafeChannelConfiguration_t *channelFailsafeConfiguration;
-    const rxFailsafeChannelConfiguration_t *channelFailsafeConfigurationDefault;
-    bool equalsDefault;
-    bool requireValue;
     for (uint32_t channel = 0; channel < MAX_SUPPORTED_RC_CHANNEL_COUNT; channel++) {
-        channelFailsafeConfiguration = &rxConfig()->failsafe_channel_configurations[channel];
-        channelFailsafeConfigurationDefault = &defaultRxConfig->failsafe_channel_configurations[channel];
-        equalsDefault = channelFailsafeConfiguration->mode == channelFailsafeConfigurationDefault->mode
+        const rxFailsafeChannelConfiguration_t *channelFailsafeConfiguration = &rxConfig->failsafe_channel_configurations[channel];
+        const rxFailsafeChannelConfiguration_t *channelFailsafeConfigurationDefault = &defaultRxConfig->failsafe_channel_configurations[channel];
```

Comment:
This one is never called with NULL `defaultRxConfig`. I guess it should be checked anyway.